configure.ac

```diff
@@ -147,6 +147,12 @@ AC_ARG_WITH([swift-toolchain],
    case $target_os in
       linux*)
 	    os_string="linux"
+	    case $target_cpu in
```

Comment:
How would libtool know what convention Swift uses for the architecture directory?